recipes/aws-iam-authenticator/LICENSE.txt

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
I've copied the License file from the original repo: https://github.com/kubernetes-sigs/aws-iam-authenticator/blob/master/LICENSE

Is this the recommended way?

Reply:
If the license is already being packaged with the software, it is not necessary

Reply:
I can see that the linter complained. Should I remove the LICENSE file and the `license_file` from the `meta.yaml` instead?

Reply:
> If the license is already being packaged with the software

It's not, so I will keep it then.

Reply:
Yes, just remove it and should be fine